doc/docs/1.10.x/deploy-manage/deploy/environment-variables.md

```diff
@@ -46,7 +46,6 @@ environment variables.
 | `PORT`                 | `650`             | The `pachd` port number. |
 | `HTTP_PORT`             | `652`             | The HTTP port number.   |
 | `PEER_PORT`             | `653`             | The port for pachd-to-pachd communication. |
-| `PPS_ETCD_PREFIX`        | `pachyderm_pps`   | ???  |
```

Comment:
Just to confirm, this is being removed cuz we don't want to document it for users right? Afaik the flag hasn't gone away. It's reasonable to not doc it for users, it only really exists for tests, and I don't think there's a good reason for users to touch it, and many good reasons for them not to touch it.

Reply:
Yes, the flag is still in the code TTBOMK. It was left accidentally in the doc.